README.md

```diff
@@ -60,29 +61,29 @@ A new group can be added via the `Add Group` button.
 
 ### Controls
 
-A new control can be added via the `Add Control` button. Beside the *Name*, *Color* and *Publish Topic* each control type has specific settings.
+A new control can be added via the `Add Control` button. Beside the _Name_, _Color_ and _Publish Topic_ each control type has specific settings.
 
 #### Button
 
-The button sends the given *Value* to the given *Publish Topic*. For more customization a icon provided by grafana can be selected.
+The button sends the given _Value_ to the given _POST PATH_. For more customization a icon provided by grafana can be selected.
 
 ![](/doc/button.png)
 
 #### Switch
 
-The switch sends the given *Value On* on true and then given *Value Off* on false. Switches also listen to the given *Subscribe Topic* and read there state from given *Listen Path*. This path should be a valid json path and the value should be boolean convertable.
+The switch sends the given _Value On_ on true and then given _Value Off_ on false. Switches also listen to the given _API Listen Path_ and read there state from given _Listen Path_. This path should be a valid json path and the value should be boolean convertable.
 
-Let's asume with get the following message from the given *Subscribe Topic*:
+Let's asume with get the following message from the given _API Listen Path_:
 
 ```json
 {
-   "switch": true,
-   "array": [true, false, false, true]
+  "switch": true,
+  "array": [true, false, false, true]
 }
 ```
 
-With a *Listen Path* of `switch` the state would be true.
-With a *Listen Path* of `array[1]` the state would be false.
+With a _Listen Path_ of `switch` the state would be true.
```

Comment:
I would recommend 
"If the *Listen Path* is 'switch [...]'" or
"With 'switch' as *Listen Path* [...]" or something like that.